Model/PayPalManagement.php

```diff
@@ -150,8 +150,14 @@ public function savePaymentAndPlaceOrder($cartId,$deviceFingerprint)
             }
         } catch (\Exception $e) {
             $this->response->setSuccess(false);
+            $error = $e->getMessage();
+            if ($error == "Blacklisted" || $error == "FraudSuspected") {
+                $errorMessage = ': PaymentAuthorizationFailed';
```

Comment:
Are the `':'` necessary? I think when we concat the string on line `160` we'll end up with double `:`